koa-crud/src/graphql/typeDefs.ts

```diff
@@ -0,0 +1,36 @@
+import { gql } from 'apollo-server-koa';
+
+export default gql`
+  type Query {
+    vendors: ArrayOfVendorReturn
+    vendor(id: String): VendorReturn
+  }
+
+  type Mutation {
+    createVendor(name: String, type: String): VendorReturn
+    updateVendor(id: String, name: String, type: String): VendorReturn
+    deleteVendor(id: String): DeleteVendorReturn
```

Comment:
To all mutations please return `true`

The reason for this is to add validation either `true` or `Error`

Reply:
noted sir will update on my next PR